src/v2/Apps/Artist/Routes/Overview/Utils/renderOrRedirect.tsx

```diff
@@ -39,17 +39,22 @@ export function renderOrRedirect({
 
   const hasArtistContent = hasOverviewContent(artist)
 
-  const alreadyAtWorksForSalePath = pathname.includes(
-    `${artist.slug}/works-for-sale`
-  )
-
+  const onOverviewPath = pathname.endsWith(`${artist.slug}`)
   const canShowOverview = showArtistInsights || hasArtistContent
 
+  const onAuctionResultsPath = pathname.includes(
+    `${artist.slug}/auction-results`
+  )
+  const canShowAuctionResults = artist?.counts?.auctionResults
+
   if (pathname === `/artist/${artist.slug}/`) {
     throw new RedirectException(`/artist/${artist.slug}`, 301)
   }
 
-  if (!canShowOverview && !alreadyAtWorksForSalePath) {
+  if (
+    (onOverviewPath && !canShowOverview) ||
+    (onAuctionResultsPath && !canShowAuctionResults)
```

Comment:
I'm not 100% sure if this covers all cases in which we want to redirect to `/works-for-sale` or if we have to add more cases.